README.md

```diff
@@ -1,88 +1,89 @@
-# CAR Mirror over HTTP v0.1.0
+# CAR Mirror over HTTP v0.2.0
 
 ## Authors
 
 * [Brooklyn Zelenka], [Fission]
 
 ## Dependencies
 
+- [CAR Mirror]
 - [CARv1]
 - [CBOR]
```

Comment:
By the way - I'd recommend linking IPLD Dag-CBOR here, since I assume the `Link`s (i.e. CIDs) inside the CBOR are supposed to be encoded in binary, and IPLD's dag-CBOR introduces its own tag for CIDs (tag 42), whereas CBOR itself doesn't specify a tag 42 or how to encode CIDs.

But that's just a comment, not really related to this PR.

Reply:
Good point 👍 Will fix

Reply:
Done